packages/app/src/components/uiBlocks/Button.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <button
+    class="bg-black cursor-pointer rounded-md text-center p-2 text-white shadow-lg"
+    @click="doClick"
+  >
+    <span class="flex">
+      <p class="my-auto font-semibold">{{ text }}</p>
+      <img v-if="imagePath" class="w-8 h-8 ml-4" :src="imagePath" />
+    </span>
+  </button>
+</template>
+
+<script>
+import { defineComponent } from 'vue';
+import { useRouter } from 'vue-router';
+
+export default defineComponent({
+  props: {
+    text: {
+      type: String,
+      required: true,
+    },
+    to: {
+      type: [Object, String],
+      default: null,
+    },
+    back: {
```

Comment:
I would rename this to clarify what this does, like
```suggestion
    revert: {
```

Reply:
@tcoro is changed :+1: